dist/platforms/windows/build.ps1

```diff
@@ -66,6 +66,21 @@ else
     Get-ChildItem -Path $Env:UNITY_PROJECT_PATH\Assets\Editor -Recurse
 }
 
+#
+# Create Android Keystore if available
+#
+
+if ( "$Env:BUILD_TARGET" -eq "Android" -and -not ([string]::IsNullOrEmpty("$Env:ANDROID_KEYSTORE_NAME")) -and -not ([string]::IsNullOrEmpty("$Env:ANDROID_KEYSTORE_BASE64")) )
+{
+    Write-Output "Creating Android keystore."
+    $keystorePath = "$Env:GITHUB_WORKSPACE\$Env:ANDROID_KEYSTORE_NAME"
+    [System.IO.File]::WriteAllBytes($keystorePath, [System.Convert]::FromBase64String($Env:ANDROID_KEYSTORE_BASE64))
+    Write-Output "Created Android keystore."
+}
+else {
+    Write-Output "Not creating Android keystore."
+}
```

Comment:
This should probably move into the TypeScript logic. We don't want to keep adding complexity in the (power)shell scripts.

Reply:
Should this be removed from the Linux build script as well and be done purely in js?

Reply:
This matches how its currently done for the Linux build script.

I think it's better to first make the Windows and Linux build scripts equivalent, and then we can go back and change both at the same time.

Reply:
Yes ultimately this is logic not passing variables to the editor. All logic is intended to be in the TS part.

In my opinion we shouldn't first make it equivalent, because that would just make it more work for me (not to mention impossible to test) when porting it over to https://github.com/game-ci/cli.

Reply:
I'll look into moving it to ts. Hopefully its relatively straightforward being a base64 conversion and a write to a file

Reply:
Implemented in latest push